cognitedata/reveal
# fix: mobile camera controls not working in certain scenarious

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-758

## Description :pencil:

Mobile camera controls were not working in Reveal documentation and swipes on screen were initiating scrolling or zooming of the page instead of navigation within Reveal window. This issue was introduced after transition to `PointerEvents` both for touch and mouse actions. Previously touch scroll was prevented inside `TouchStart` event but the same logic is not possible with `PointerEvents`. Also, this PR adds capability for more "variations" within example page for more diversified manual testing. 

## How has this been tested? :mag:

In chrome devTools mobile device emulator with local version of Reveal documentation.



## Test instructions :information_source:
1. Open documentation preview link on mobile device and try to navigate within any loaded model.
2. Open currently deployed [master documentation](https://cognitedata.github.io/reveal-docs/docs/next/examples/cad-basic) and do the same thing.
3. Notice that master documentation doesn't allow to navigate around the model.



## Checklist :ballot_box_with_check:
